web/libs/frontend-test/src/helpers/LSF/AudioView.ts

```diff
@@ -36,7 +37,9 @@ export const AudioView = {
     return this.root.get("loading-progress-bar", { timeout: 10000 });
   },
   isReady() {
+    LabelStudio.waitForObjectsReady();
     this.loadingBar.should("not.exist");
+    cy.wait(32); // wait for render
```

Comment:
can we use any kind of observer to wait for actual audio rendered? 32s is too long...

Reply:
But yeah, we can add some code into `AudioUltra` to make `isReady` work better, but for now it feels like too much, as we did not add it when it was necessary. And right now I have no free time in the boundaries of estimations for this task.